runtime/module.go

```diff
@@ -202,3 +205,7 @@ func ProvideTransientStoreService(key depinject.ModuleKey, app *AppBuilder) stor
 	storeKey := ProvideTransientStoreKey(key, app)
 	return transientStoreService{key: storeKey}
 }
+
+func ProvideEventService(ctx context.Context, app *AppBuilder) event.Service {
```

Comment:
```suggestion
func ProvideEventService() event.Service {
```
This doesn't require any parameters